_posts/2021-03-05-scala-3-developers-preview-survey-results.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: blog-detail
+post-type: blog
+by: Scala Center
+title: Scala 3 Developer’s Preview Survey Results
+---
+## Context
+
+After 8 years of development, the Dotty project is well on its way to become the new Scala compiler,
+known in the community as Scala 3.
+
+Since announcing this news (ScalaDays Berlin 2018), it has taken a good 2+ years to finalize the changes,
+sync the community’s efforts, publish migration paths, and create a new ecosystem of tools and libraries.
+We are very near to being able to take the final step - releasing Scala 3.0.
+We are still not there yet but are closer than ever.
+
+In December 2020 Scala 3.0.0-M3 was released, the last milestone before the RC1 release, scheduled for January 2021.
+
+Scala 3 release teams were ready to move forward but still needed
+a final feedback loop from the community to do so. We published a blog on December 15th 2020,
+sharing the status of the Scala 3 release, and to ask the Scala community 
+to give us their opinions, concerns and advice.
+
+Today, we are pleased to share what we learned with you.
+First, we want to thank everyone who participated and selflessly poured
+in such a rich amount of insights and knowledge our way! Based on this survey,
+we will take actions and lead conversations. So, keep following and sharing,
+the quality of Scala development depends on your feedback.
+
+Below, we will present our findings in 3 sections:
+1. _**About the survey**_ - contains all information about 
+   creating the survey and collecting data, and methodology for our analysis.
+2. _**Presenting the data**_ - summarizes the data we received, mostly quantifiable
+3. _**Conclusions**_ - this is the part that shows 
+   how we read the data and what steps we will take based on our reading.
+
+
+Conclusions are open to interpretation, which is why we offer full
+insight into our methodology and data, so feel free to make your own and share it with us.
```

Comment:
"insight into" is a bit odd here since "insight" doesn't just mean you can see in, it means understanding. I think "access to" is closer to the presumed intended meaning. or reword using "make available" or "provide openly", that kind of thing.